mpqp/core/instruction/measurement/basis.py

```diff
@@ -71,7 +87,7 @@ def __init__(
             raise ValueError("All vectors of the given basis are not normalized")
         m = np.array([vector for vector in basis_vectors])
         if not matrix_eq(
-            m.transpose().dot(m),
+            m.transpose().dot(m), #fixme
```

Comment:
good catch, it should be fixed in bb17fef

Reply:
note: there was a few errors in the commit, it's fixed in the following ones, but having a look at it will already give you a good indication of the fix

Reply:
fixed for real

Reply:
Ok good